resources/views/account/accept-asset.blade.php

```diff
@@ -2,59 +2,142 @@
 @section('content')
 
+    
+<link rel="stylesheet" href="{{ asset('assets/css/signature-pad.css') }}">
+
 <style>
 .form-horizontal .control-label, .form-horizontal .radio, .form-horizontal .checkbox, .form-horizontal .radio-inline, .form-horizontal .checkbox-inline {
     padding-top: 17px;
     padding-right: 10px;
 }
 
-.radio input[type="radio"], .radio-inline input[type="radio"], .checkbox input[type="checkbox"], .checkbox-inline input[type="checkbox"] {
-    margin-left: -40px;
+#eula_div {
+    width: 100%;
+    height: 200px;
+    overflow: scroll;
 }
+
 </style>
-<div class="row header">
-    <div class="col-md-12">
-
-        <h3>
-        Accept {{ $item->showAssetName() }}</h3>
-    </div>
-</div>
-
-
-<div class="col-md-12">
-        <form class="form-horizontal" method="post" action="" autocomplete="off">
-        <!-- CSRF Token -->
-        <input type="hidden" name="_token" value="{{ csrf_token() }}" />
-        <input type="hidden" name="logId" value="{{ $findlog->id }}" />
-
-        <div class="radio">
-          <label>
-            <input type="radio" name="asset_acceptance" id="accepted" value="accepted"> I accept
-          </label>
-        </div>
-
-        <div class="radio">
-          <label>
-            <input type="radio" name="asset_acceptance" id="declined" value="declined"> I decline
-          </label>
-        </div>
-
-
-        <!-- Form actions -->
-        <div class="form-group">
-            <div class="col-md-7 col-md-offset-3">
-                <button type="submit" class="btn btn-success">Submit </button>
-            </div>
-        </div>
-    </form>
-    </div>
-</div>
+
+
+<form class="form-horizontal" method="post" action="" autocomplete="off">
+    <!-- CSRF Token -->
+    <input type="hidden" name="_token" value="{{ csrf_token() }}" />
+    <input type="hidden" name="logId" value="{{ $findlog->id }}" />
+
+
+<div class="row">
+    <div class="col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2">
+
+        <div class="panel box box-default">
+
+            <div class="box-body">
+                <div class="col-md-12">
+
+
+                <div class="radio">
+                    <label>
+                        <input type="radio" name="asset_acceptance" id="accepted" value="accepted">
+                        I accept
+                    </label>
+                </div>
+                <div class="radio">
+                    <label>
+                        <input type="radio" name="asset_acceptance" id="declined" value="declined">
+                        I decline
+                    </label>
+                </div>
+
+                @if ($item->getEula())
+                <div class="col-md-12" style="padding-top: 20px">
+                    <div id="eula_div">
+                        {!!  $item->getEula() !!}
+                    </div>
+                 </div>
+                @endif
+
+                @if (\App\Models\Setting::getSettings()->require_accept_signature=='1')
```

Comment:
Easy enough to do via middleware, but I think we'd still need something like $request->settings->blah, which doesn't seem shorter. We cache the value of the settings query at least, so we're not running it every time we need a setting.